t/op/magic.t

```diff
@@ -433,6 +433,22 @@ EOP
   }
 }
 
+# Check that assigning to $0 properly handles upgraded strings:
+{
+  my $eacute = "é";
```

Comment:
It would be more robust to use `\N{U+00E9}` and not depend on whether "use utf8" is used in this scope.

Reply:
Rather, the UTF-8 encoding of it since that's what we want here. 0xC3 0xA9

Reply:
Fixed. Also fixed the test count.